be/src/runtime/current_thread.h

```diff
@@ -95,7 +97,6 @@ class CurrentThread {
 
         bool try_mem_consume_with_limited_tracker(int64_t size, MemTracker* tracker, int64_t limit) {
             MemTracker* cur_tracker = _loader();
-            size = _consume_from_reserved(size);
```

Comment:
why remove this line?

Reply:
Seems like unused logic, I just deleted it